docs/man/man1/tkn-results-config-set.1

```diff
@@ -2,50 +2,46 @@
 .TH "TKN-RESULTS" "1" "Aug 2025" "Tekton Results CLI" ""
 
 .SH NAME
-tkn-results-config-set - Configure Tekton Results CLI settings
+tkn-results-config-set - Set Tekton Results CLI configuration values
 
 
 .SH SYNOPSIS
 \fBtkn-results config set\fP
 
 
 .SH DESCRIPTION
-Configure settings for the Tekton Results CLI.
+Configure how the CLI connects to the Tekton Results API server.
 
 .PP
-This command allows you to configure how the CLI interacts with the Tekton Results API server.
-It can automatically detect the API server in your cluster or allow manual configuration.
+Usage Modes:
+1. Interactive: Prompts for values with defaults where available
+   tkn-results config set
+.IP "  1." 5
+Manual: Specify values via flags
+tkn-results config set --host= --token= --api-path=
 
 .PP
-The command will:
-1. Automatically detect the Tekton Results API server in your cluster
-2. Prompt for any missing configuration values
-3. Save the configuration for future use
-
-.PP
-Automatic Detection:
-- Cluster context and namespace
-- API server endpoint
-- Service account token (if available)
+Configuration Options:
+- Host: Tekton Results API server URL
+- Token: Bearer token (defaults to current kubeconfig token)
+- API Path: API endpoint path
+- TLS Settings: Certificate verification options
 
 .PP
-Manual Configuration (if automatic detection fails):
-- API server host (e.g., http://localhost:8080)
-- Authentication token
-- Additional cluster settings
+Use manual configuration when:
+- Route is not in openshift-pipelines namespace
+- Route name differs from tekton-results-api-service
+- Using custom domain patterns
+- On Kubernetes clusters (ingress hostnames vary)
 
 .PP
-Configuration Options:
-  --host                    API server host URL
-  --token                   Authentication token
-  --api-path                API server path prefix
-  --insecure-skip-tls-verify Skip TLS certificate verification
-  --kubeconfig, -k          Path to kubeconfig file
-  --context, -c             Kubernetes context to use
-  --namespace, -n           Kubernetes namespace
+Route Requirements (OpenShift):
+- Route name: tekton-results-api-service
+- Namespace: openshift-pipelines
+- Expected URL format: https://-\&.apps.
```

Comment:
~I've never seen this format specification before. Can you clarify the format here with regex?~

I think since this line isn't a code block in the markdown the string `https://<route-name>-<namespace>.apps.<cluster-domain>` is being misconverted here in the manpage as `https://-\&.apps.`

Reply:
Nice catch! Fixed.